app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Suyabay\Http\Controllers;
+
+use Suyabay\User;
+use Suyabay\Role;
+use Suyabay\Invite;
+use Suyabay\Http\Requests;
+use Illuminate\Http\Request;
+use Illuminate\Mail\Mailer as Mail;
+use Illuminate\Database\QueryException;
+use Suyabay\Http\Controllers\Controller;
+
+class UserController extends Controller
+{
+    protected $mail;
+    protected $response;
+
+    public function __construct(Mail $mail)
+    {
+        $this->mail = $mail;
+    }
+
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        $users = User::orderBy('id', 'asc')->paginate(10);
+
+        return view('dashboard.pages.user', compact('users'));
+    }
+
+    /**
+     * Check if user exist
+     *
+     * @param  $request
+     */
+    public function checkIfUserExist(Request $request)
+    {
+        $user = User::where('username', $request->username)->first();
+
+        if ($user)
+        {
+           return true;
+        }
+    }
+
+    /**
+     * Insert invitation into database
+     *
+     * @param  $request
+     */
+    public function createInvitation(Request $request)
+    {
+        try
+        {
+            Invite::create([
+                'username'  => $request->username,
+                'role_id'  => $request->user_role,
+                'token'   => $request->_token
+            ]);
+            $this->response =
+            [
+                'message' => 'Invitation created',
+                'status_code' => 201
+            ];
+        } catch (QueryException $e) {
+            $this->response =
+            [
+                'message' => 'Invitation already sent',
+                'status_code' => 400
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Check if user exist and Insert Invitation details
+     * into the database
+     *
+     * @param  $request
+     */
+    public function insertInvite(Request $request)
+    {
+        if ( $this->checkIfUserExist($request) )
+        {
+            $this->response = $this->createInvitation($request);
+        }
+        else
+        {
+            $this->response =
+            [
+                'message' => 'User does not exist',
+                'status_code' => 400
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Process invitation details into invites table
+     *
+     * @param  Request $request
+     */
+    public function createInvite(Request $request)
+    {
+        $createInvite = $this->insertInvite($request);
+
+        if ($createInvite['status_code'] == 201)
+        {
+            $this->response = $this->getInviteeEmail($request);
+        }
+        else
```

Comment:
The opening brace should on the same line as the `if` and the `else` should be on the same line as the closing `if` brace. Check out the PSR-2 guide [http://www.php-fig.org/psr/psr-2/](http://www.php-fig.org/psr/psr-2/)